.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
![Invalid](https://user-images.githubusercontent.com/93039359/198346281-7ab63e89-4171-48b7-a3f4-9615e4b4a559.PNG)

- [ ] Your linters are not passing all the tests. Kindly fix the problems to pass the tests because linters detect those errors which can break your application 🤔